atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellLoader.java

```diff
@@ -92,8 +95,20 @@ void loadWithTs(
             boolean loadAllTs,
             CassandraKeyValueServices.ThreadSafeResultVisitor visitor,
             ConsistencyLevel consistency) {
-        Map<InetSocketAddress, List<Cell>> hostsAndCells =
-                HostPartitioner.partitionByHost(clientPool, cells, Cell::getRowName);
+        Map<InetSocketAddress, List<Cell>> hostsAndCells;
+        try (CloseableTracer tracer = CloseableTracer.startSpan(
+                "partitionByHost",
+                ImmutableMap.of(
+                        "cells",
+                        String.valueOf(cells.size()),
+                        "tableRef",
+                        Objects.requireNonNull(LoggingArgs.tableRef(tableRef).getValue()),
```

Comment:
Hmm, I was expecting something like `safeInternalTableNameOrPlaceholder` actually. If I'm not mistaken that would actually spit out the table's name regardless.

(Internally there's the assumption that table names in this KVS are safe, but still...)

Reply:
Fixed.